pages/docs/some-react-code/index.jsx

```diff
@@ -3,18 +3,18 @@ import Demo from './_Demo'
 import DocumentTitle from 'react-document-title'
 import { config } from 'config'
 
+exports.data = {
+  title: 'Some React Code',
+  date: `${new Date()}`
```

Comment:
Just change it to a static date (ISO 8601) and I'll commit this. In any case, it doesn't make sense to have a dynamic date in this context. Actually... just drop the date altogether. A documentation page wouldn't need it.